pom.xml

```diff
@@ -121,6 +121,12 @@
             <version>${jacoco.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-api</artifactId>
+            <version>1.7.30</version>
+            <scope>compile</scope>
```

Comment:
This in compile scope, because tddb and exasol-containers have this already?

Reply:
No idea. I just copied it from virtual schemas...